MAINTAINERS.md

```diff
@@ -10,3 +10,4 @@
 | David Powers | [dapowers87](https://github.com/dapowers87) | Amazon |
 | Shivani Shukla | [sshivanii](https://github.com/sshivanii) | Amazon |
 | David Venable | [dlvenable](https://github.com/dlvenable) | Amazon |
+| Phill Treddenick | [treddeni-amazon](https://github.com/treddeni-amazon) | Amazon |
```

Comment:
Currently these are listed alphabetically by last name. It might be good to stick with that.